tests/test_client_regress/views.py

```diff
@@ -151,3 +151,17 @@ def render_template_multiple_times(request):
     """A view that renders a template multiple times."""
     return HttpResponse(
         render_to_string('base.html') + render_to_string('base.html'))
+
+def redirect_based_on_headers_1_view(request):
+    """A view that redirects iff HTTP_REDIRECT is set"""
+    if "HTTP_REDIRECT" in request.META:
```

Comment:
Use single quotes (unless a string contains a single quote) as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).